Cargo.toml

```diff
@@ -65,17 +65,5 @@ lto = "thin"
 
 [patch.crates-io]
 
-# TODO: replace with upstream librustzcash when these changes are merged (#3037)
-#       remove these after a new librustzcash release (#2982)
-
-# These are librustzcash file requirements specified in its workspace Cargo.toml,
-# that we must replace with git requirements
-equihash = { git = "https://github.com/zcash/librustzcash.git", rev = "d14e7a707ce01cefcbc82651dad48f002185dded" }
-hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "9b4c1bdbe0517e3a7a8f285d6048a37d472ba3bc" }
-zcash_encoding = { git = "https://github.com/zcash/librustzcash.git", rev = "d14e7a707ce01cefcbc82651dad48f002185dded" }
-zcash_note_encryption = { git = "https://github.com/zcash/librustzcash.git", rev = "d14e7a707ce01cefcbc82651dad48f002185dded" }
-zcash_primitives = { git = "https://github.com/zcash/librustzcash.git", rev = "d14e7a707ce01cefcbc82651dad48f002185dded" }
-# Currently pointing to `download-sprout-params-new` branch which contains the
-# changes from `download-sprout-params-d14e7a70` rebased on top of d14e7a70 as
-# required by `zcash/Cargo.toml`.
-zcash_proofs = { git = "https://github.com/ZcashFoundation/librustzcash.git", rev = "c3ff94134b472ba90af665e5454983dc12de1338" }
+# Currently pointing to `download-sprout-params` branch.
+zcash_proofs = { git = "https://github.com/ZcashFoundation/librustzcash.git", rev = "6d75718076e592a41b6bd6ec916dc15420e4cc3c" }
```

Comment:
We can leave this as is if needed, but ideally we'd wait for https://github.com/zcash/librustzcash/pull/459 to be merged.

Reply:
I think we need to merge without this but if it gets merged before rc.0 we can pull it in then

Reply:
I don't think it's going to merge in the next few days, there hasn't been any progress on the review.